faiss/IndexIDMap.cpp

```diff
@@ -120,7 +120,6 @@ void IndexIDMapTemplate<IndexT>::search(
     }
     index->search(n, x, k, distances, labels, params);
     idx_t* li = labels;
-#pragma omp parallel for
```

Comment:
Why not make this ..

```
#pragma omp parallel for if (n * k > 1)
```

.. as well?

Reply:
n is always 1 ...
k is the K value in the query vector

so n * k is always > 1

the reason why i removed this pragma was that
in systems with large number of vCPUs, we can spawn a very large number
of threads for doing a simple operation here

Reply:
I see, ok in that case - let's add commentary here and everywhere you're making changes citing are reasons.

Very soon, we'll likely be considering merging the latest from upstream into our fork and we shouldn't accidentally undo any of our changes.

Reply:
added